src/cag/visualization.py

```
@@ -0,0 +1,807 @@
+            'constraint_level': zoning.get('constraint_level', 'Uncategorized'),
+            'development_gap': zoning.get('development_gap', 0),
+            'opportunity_score': reform.get('opportunity_score', 0),
+            'permit_utilization': reform.get('permit_utilization', 0),
+        })
+
+    return sorted(zip_data, key=lambda x: x['growth_score'], reverse=True)
+
+
+def _generate_html(
+    zip_data: List[Dict[str, Any]],
+    multifamily_data: Dict[str, Any],
```

Ruff (0.14.8)

Unused function argument: `multifamily_data`

(ARG001)